src/content/en/updates/2017/02/lighthouse-jan.md

```diff
@@ -0,0 +1,147 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: What's new in Lighthouse 1.5.
+
+{# wf_updated_on: 2017-02-10 #}
+{# wf_published_on: 2017-02-10 #}
+{# wf_tags: lighthouse,tools,performance,progressive-web-apps #}
+{# wf_featured_image: /web/progressive-web-apps/images/pwa-lighthouse.png #}
+{# wf_featured_snippet: What's new in Lighthouse 1.5. New audits, extension updates, Performance Experiment, online Viewer features, and UI tweaks. #}
+
+# Lighthouse January 2017 update {: .page-title }
+
+{% include "web/_shared/contributors/brendankenny.html" %}
+{% include "web/_shared/contributors/ericbidelman.html" %}
+
+[Lighthouse](/web/tools/lighthouse/) is an
+[open-source](https://github.com/GoogleChrome/lighthouse), automated tool for
+improving the quality of your web apps. You can install it as a
+[Chrome Extension][crx] or run it as a Node command line tool. When you
+give Lighthouse a URL, it runs a barrage of tests against the page and then
```

Comment:
Mostly a cut and paste from the previous post and https://developers.google.com/web/tools/lighthouse/